### PR TITLE
[Chore](materialized-view) output reference column info when create mv can't find ref column

### PR DESCRIPTION
**Pull request by BiteTheDDDDt**
## Proposed changes

2023-11-17 12:29:40,276 WARN (thrift-server-pool-0|217) [MasterImpl.finishTask():94] finish task reports bad. request: TFinishTaskRequest(backend:TBackend(host:172.16.0.84, be_port:9061, http_port:8041), task_type:ALTER, signature:128361, task_status:TStatus(status_code:INTERNAL_ERROR, error_msgs:[(172.16.0.84)[INTERNAL_ERROR]rollup job meet invalid ref_column, new_column=mv_k1]), report_version:17001953500000)
2023-11-17 12:29:40,276 WARN (thrift-server-pool-2|251) [MasterImpl.finishTask():94] finish task reports bad. request: TFinishTaskRequest(backend:TBackend(host:172.16.0.84, be_port:9061, http_port:8041), task_type:ALTER, signature:128363, task_status:TStatus(status_code:INTERNAL_ERROR, error_msgs:[(172.16.0.84)[INTERNAL_ERROR]rollup job meet invalid ref_column, new_column=mv_k1]), report_version:17001953500000)
2023-11-17 12:29:40,276 WARN (thrift-server-pool-1|250) [MasterImpl.finishTask():94] finish task reports bad. request: TFinishTaskRequest(backend:TBackend(host:172.16.0.84, be_port:9061, http_port:8041), task_type:ALTER, signature:128365, task_status:TStatus(status_code:INTERNAL_ERROR, error_msgs:[(172.16.0.84)[INTERNAL_ERROR]rollup job meet invalid ref_column, new_column=mv_k1]), report_version:17001953500000)
2023-11-17 12:29:40,330 INFO (materialized view|30) [RollupJobV2.runRunningJob():519] rollup tasks not finished. job: 128359
2023-11-17 12:29:40,330 WARN (materialized view|30) [RollupJobV2.runRunningJob():525] rollup task failed: task type: ALTER, status_code: INTERNAL_ERROR, status_message: [(172.16.0.84)[INTERNAL_ERROR]rollup job meet invalid ref_column, new_column=mv_k1], backendId: Backend [id=10003, host=172.16.0.84, heartbeatPort=9051, alive=true, lastStartTime=2023-11-17 12:29:31, process epoch=1700195371566, tags: {location=default}], signature: 128361

2023-11-17 12:29:40,330 INFO (materialized view|30) [Partition.deleteRollupIndex():203] delete shadow rollup index 128360 in partition 120915-agg_have_dup_base_no_await
2023-11-17 12:29:40,331 INFO (materialized view|30) [OlapTable.deleteIndexInfo():439] delete index info k12s3m in table 120916-agg_have_dup_base_no_await
2023-11-17 12:29:40,339 INFO (materialized view|30) [RollupJobV2.cancelImpl():640] cancel ROLLUP job 128359, err: errCode = 2, detailMessage = rollup tasks failed on same tablet reach threshold [[ALTER], signature: 128361, backendId: 10003, tablet id: 128361], reason=task type: ALTER, status_code: INTERNAL_ERROR, status_message: [(172.16.0.84)[INTERNAL_ERROR]rollup job meet invalid ref_column, new_column=mv_k1], backendId: Backend [id=10003, host=172.16.0.84, heartbeatPort=9051, alive=true, lastStartTime=2023-11-17 12:29:31, process epoch=1700195371566, tags: {location=default}], signature: 128361
2023-11-17 12:29:40,339 INFO (materialized view|30) [MaterializedViewHandler.onJobDone():1165] set table's state to NORMAL, table id: 120916, job id: 128359

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

File: be/src/olap/schema_change.cpp [hunk 1]
```diff
@@ -1191,17 +1186,14 @@ Status SchemaChangeHandler::_parse_request(const SchemaChangeParams& sc_params,
         ColumnMapping* column_mapping = changer->get_mutable_column_mapping(i);
         column_mapping->new_column = &new_column;
 
-        if (materialized_function_map.find(column_name) != materialized_function_map.end()) {
-            auto mvParam = materialized_function_map.find(column_name)->second;
-            column_mapping->expr = mvParam.expr;
-            int32_t column_index = base_tablet_schema->field_index(mvParam.origin_column_name);
-            if (column_index >= 0) {
-                column_mapping->ref_column = column_index;
+        if (materialized_function_map.find(to_lower(column_name)) !=
```
Comment (HappenLee) [inline be/src/olap/schema_change.cpp:1189]: if need lower column name，only call once `to_lower` before the function

**Comment by BiteTheDDDDt** [comment]
run buildall

**Review by HappenLee — APPROVED** [review]
LGTM

**Review by Mryange — APPROVED** [review]

**Pull request merged**